source/Test.cpp

```diff
@@ -0,0 +1,100 @@
+/* Test.cpp
+Copyright (c) 2019 by Peter van der Meer
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+*/
+
+
+#include "DataNode.h"
+#include "PlayerInfo.h"
+#include "Test.h"
+#include "TestStep.h"
+#include "UI.h"
+#include <string>
```

Comment:
You already have `<string>` in the header (and these includes are not ordered properly)